SlevomatCodingStandard/Helpers/ClassHelper.php

```diff
@@ -18,4 +18,22 @@ public static function getName(\PHP_CodeSniffer_File $codeSnifferFile, int $clas
 		return $tokens[$codeSnifferFile->findNext(T_STRING, $classPointer + 1, $tokens[$classPointer]['scope_opener'])]['content'];
 	}
 
+	public static function getAllNames(\PHP_CodeSniffer_File $codeSnifferFile): array
```

Comment:
Could you please provide simple test for this method? All helper methods have tests too, see `ClassHelperTest`.